### PR TITLE
Features/alert buttons

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Completed alert and button component.

### CODE REVIEW

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+.alert-success {
+  background-color: #4ade80;
+  border-radius: 0.2em;
+  color: #14532d;
+}
+.alert-warning {
+  background-color: #fde68a;
+  border-radius: 0.2em;
+  color: #e09004;
```
Comment (bishnoimukesh) [inline Components/alert.css]: use single style for color eg: if you are using hex code than use that at all places

File: Components/button.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
+.bttn {
```
Comment (bishnoimukesh) [inline Components/button.css:1]: use btn or button instead of bttn for readability

**Review by bishnoimukesh — CHANGES_REQUESTED** [review]
few changes are suggested to look over rest all good

File: Components/alert.css (hunk 1 above)
Comment (Rishi05051997) [reply Components/alert.css]: you can use variables for giving colors put that variable inside :root

**Review by Rishi05051997 — COMMENTED** [review]
other things are ok

File: Components/alert.css (hunk 1 above)
Comment (SHIKHASINGH1506) [reply Components/alert.css]: Added variables after review. Thanks for the review!

**Review by devdeadviz — COMMENTED** [review]
looks go to me

File: Components/alert.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  background-color: var(--light-danger-color);
+  border-radius: 0.2em;
+  color: var(--danger-text-color);
+}
+.alert-icon {
+  margin-right: 0.5em;
+}
+.w-100 {
+  width: 50%;
+}
```
Comment (h-harsh) [inline Components/alert.css:38]: can be added in utility class, and proper naming

File: utility.css [hunk 4]
```diff
@@ -1,114 +1,116 @@
 /* padding utility */
-.p-2{
-    padding: 0.5rem;
+.p-2 {
+  padding: 0.5rem;
 }
```
Comment (h-harsh) [inline utility.css:4]: using a proper scale

**Review by h-harsh — COMMENTED** [review]
Very minor changes needs to be done

**Pull request merged**